# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bdiff/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

/usr/include/gnu/stubs-32.h:7:3: error: #error Applications may not define the macro _LIBC

  - Rene Rebe <rene@exactcode.de>

--- bdiff-1.0.5/Makefile.vanilla	2007-01-03 13:38:29.000000000 +0000
+++ bdiff-1.0.5/Makefile	2007-01-03 13:38:38.000000000 +0000
@@ -14,7 +14,7 @@
 	strip bdiff
 
 lib/md5.o:
-	cc -O2 -D_LIBC -c lib/md5.c -o lib/md5.o
+	cc -O2 -c lib/md5.c -o lib/md5.o
 
 clean:
 	rm -f bdiff search.o lib/md5.o
